data/datafiles.json

```diff
@@ -314,6 +314,13 @@
             "filename": "hello-0200",
             "size": "166 MB",
             "url": "http://yt-project.org/data/hello-0200.tar.gz"
+        },
+        {
+            "code": "Enzo-P",
+            "description": "The initial dataset from a 3D simulation of a collapsing tophat with both particles and gas.",
+            "filename": "collapse-3d-0000",
+            "size": "108 MB",
```

Comment:
Looks like the dataset on the webserver is only 442 KB.

Reply:
It tars out to 108 MB.  There are lots of field values that are the same everywhere, so it zips down a lot.

Reply:
I think we only care about the zipped size.

Reply:
Alright then.  I guess I always thought it was the other way since the size of the final dataset seems like something one would want to know.  I'll change it, though.

Reply:
My logic is that when you're on that page you probably care more about the bandwidth cost than the hard drive cost, especially if you're on crappy internet or in a place with bad connectivity to the US.

I wouldn't be surprised if there are inconsistencies. We should probably choose one or the other and enforce that.